pkg/apis/workflow/v1alpha1/workflow_types.go

```diff
@@ -1005,6 +1005,11 @@ func (n *NodeStatus) IsResolvable() bool {
 	return true
 }
 
+// IsActiveSuspendNode returns whether this node is an active suspend node
+func (n *NodeStatus) IsActiveSuspendNode() bool {
```

Comment:
Can a node be non-running and suspended? If so, how about `IsSuspened`? If not, how about `IsRunningSuspended`?

Reply:
Yes. Say a workflow looks like this:
```
A -- B -- C
```
where B is a suspend node. After a user resumes the workflow, B's status would become `Succeeded`.

Another thing I'm not sure about is wording. I used `Suspend` here based on the node type `Suspend` to indicate whether this node is a suspend node or not. If we use Suspended here, it sounds like the node itself is suspended, which is not true.

Reply:
I think `IsCurrentlySuspended` is good... it should be clear from the code and comments that this only applies to nodes of type `Suspend`

Reply:
`IsCurrentlySuspended` sounds more like a workflow's method to me. I believe we can make it clear enough with comments; I just want to make it more easy to understand. I think when others read `node.IsCurrentlySuspended`, they might
- first wonder whether a node can be suspended
- and then find out it actually only applies to suspend nodes after reading the comment

Or maybe they might not. Do you think my concern is legit?

Reply:
Hmmm, I think I came around on `IsActiveSuspend`. We can leave this as is